go.work

```diff
@@ -1,13 +1,15 @@
-go 1.19
+go 1.20
+
```

Comment:
keep the go version, do not upgrade here

Reply:
Done, this was cause by running `go work` sync To solve this we ran `go work sync` and `go work edit -go=1.19`